test/manifests/double.m3u8

```diff
@@ -0,0 +1,6 @@
+# A simple main playlist with 2 media playlists
+#EXTM3U
+#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=240000,RESOLUTION=396x224
```

Comment:
We can probably remove `PROGRAM-ID` since it's no longer in the spec (same for other occurrences)